BREAKING.md

```diff
@@ -14,6 +19,73 @@
 - `FlushMode.Manual` to `FlushMode.TurnBased`
 - `FlushMode.Automatic` to `FlushMode.Immediate`
 
+### `name` removed from ContainerSchema
+The `name` property on the ContainerSchema was used for multi-container scenarios but has not materialized to be a useful schema property. The feedback has been negative to neutral and the goal is to remove it before it becomes formalized. Support for multi-container scenarios, if any is required, will be addressed as a future change.
```

Comment:
```suggestion
The `name` property on the ContainerSchema was used for multi-container scenarios but has not materialized to be a useful schema property. The feedback has been negative to neutral so it is being removed before it becomes formalized. Support for multi-container scenarios, if any is required, will be addressed as a future change.
```

Reply:
Actually this is the same note as `### TinyliciousClient and AzureClient container API changed` below right?  So it's not a 0.47 change?